config/test.exs

```diff
@@ -70,6 +70,13 @@ config :trento, Trento.Integration.Checks.Wanda.Messaging.AMQP,
     ]
   ]
 
+config :trento, Trento.Scheduler,
```

Comment:
This is needed to avoid unexpected heartbeat failure messages, as the scheduler runs the checks randomnly and as the dispatch is mocked now, it raises a Mox error.
This was the easiest solution. Other would be to not start the scheduler at all on tests, but this requires changes on how app children are passed, which at the end makes things more confusing.

Reply:
that's super ok